### DISCUSSION TITLE
How the subjective question functions or works in Question set??

### DISCUSSION THREAD

**Discussion by sachinb80** [Sunbird-Ed/Community#437] [General Discussion]
I am finding it difficult to comprehend how the subjective question in the Question set functions or works.
Can we assign marks to it?
How many languages does it support while answering the question?
Check the font of languages that will be supported or not.
Can any vernacular language content be supported while creating content like in the question set?

**Comment by Krishnaj20** [comment]
@sachinb80 - since this is related to questions, request you to start this thread in inQuiry discussion forums: https://github.com/sunbird-inquiry/community/discussions

For the moment, I'm locking this thread.
